pkg/collector/collector.go

```diff
@@ -220,6 +220,18 @@ func (c *Collector) started() bool {
 	return c.state.Load() == started
 }
 
+// MapOverRunningChecks call the callback with the list of checks locked.
```

Comment:
Is "running" here potentially confusing?  This would include all checks, not just those currently executing on a runner, right?

Reply:
No, that's the difference between AutoDiscovery and the collector. AC has all the configurations while the collector is only the check actually running. Some checks are designed to have a single instance running, so you might have more configuration that checks (ex: you configure multiple `cpu` check instance).

Reply:
What about `MapOverChecks` ?